src/Microsoft.DotNet.Interactive.Jupyter.Tests/JupyterKernelTestBase.cs

```diff
@@ -22,8 +22,8 @@ public abstract class JupyterKernelTestBase : IDisposable
 {
     protected CompositeDisposable _disposables = new();
 
-    // to re-record the tests for simulated playback with JuptyerTestData, set this to true
-    protected const bool RECORD_FOR_PLAYBACK = false;
+    // to re-record the tests for simulated playback with JupyterTestDataAttribute, set this to true
+    protected const bool RECORD_FOR_PLAYBACK = true;
```

Comment:
Curious, are we leaving this value change as is? The comment above makes me think this was meant to be false by default.

Reply:
Good point. It's not actually clear to me though that having it set to `false` is useful. If integration tests are running, why not update the recordings? 

There's probably another round of test redesign to be done here when I get more familiar with this.